src/components/Calculator.js

```diff
@@ -1,18 +1,167 @@
 import '../styles/calculator.css';
 
 export class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: 0,
+      next: null,
+      operation: null,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(btn) {
+    const btnValue = btn.target.textContent;
+    this.setState((e) => calculate(e, btnValue));
   }
 
   render() {
+    const { total, next, operation } = this.state;
     return (
       <div className="wrapper">
-        <p className="display">0 </p>
-        <KeyboardPad />
+        <div className="display">
+          {total}
+          {operation}
+          {next}
+        </div>
+        <div className="keyBoard">
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-0"
+          >
+            0
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-1"
+          >
+            1
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-2"
+          >
+            2
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className=" btn btn-3"
+          >
+            3
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className=" btn btn-4"
+          >
+            4
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-5"
+          >
+            5
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className=" btn btn-6"
+          >
+            6
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className=" btn btn-7"
+          >
+            7
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-8"
+          >
+            8
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-9"
+          >
+            9
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-Add"
+          >
+            +
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-Minus"
+          >
+            -
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-Div"
+          >
+            /
+          </button>
```

Comment:
- [ ] The "/" sign you are using on this button is not the one passed for division in the `operate.js` file and for that reason when you want to perform division, the button text content does not match the one passed in the condition in the operate.js file and hence the app crashes to a blank screen. Kindly edit the division sign in the operate.js file to have the "/" sign. 👍

Reply:
I just figured it out after I submitted it for review. It has been fixed.